api/server.go

```diff
@@ -29,18 +29,20 @@ func Serve(ctx context.Context, serveAddr string, db idb.IndexerDb, log *log.Log
 	e.Use(middlewares.MakeLogger(log))
 	e.Use(middleware.CORS())
 
+	middleware := make([]echo.MiddlewareFunc, 0)
+
+	middleware = append(middleware, middlewares.MakeMigrationMiddleware(db))
 
-	maybeAuth := make([]echo.MiddlewareFunc, 0)
 	if len(tokens) > 0 {
-		maybeAuth = append(maybeAuth, middlewares.MakeAuth("X-Indexer-API-Token", tokens))
+		middleware = append(middleware, middlewares.MakeAuth("X-Indexer-API-Token", tokens))
 	}
 
 	api := ServerImplementation{
 		EnableAddressSearchRoundRewind: developerMode,
 		db:                             db,
 	}
 
-	generated.RegisterHandlers(e, &api, maybeAuth...)
+	generated.RegisterHandlers(e, &api, middleware...)
 	common.RegisterHandlers(e, &api)
```

Comment:
@brianolson "common" is just the root endpoints, which for indexer is just /health. So I'm just not attaching the migration middleware here to allow the health endpoint to run regardless of the migration state.